code/arcaders-1-9/src/phi/mod.rs

```diff
@@ -31,15 +32,17 @@ struct_events! {
 pub struct Phi<'window> {
     pub events: Events,
     pub renderer: Renderer<'window>,
+    pub ttf_context: Sdl2TtfContext,
```

Comment:
I probably don't need to make it public actually; but I'm not sure if it's a good idea to even include it in Phi or if the font setup should be refactored a bit. I might also be missing a lifetime here, not fully familiar with those yet.